package.json

```diff
@@ -68,14 +68,14 @@
     "partitioninfo": "^6.0.2",
     "rwmutex": "^1.0.0",
     "tslib": "^2.0.0",
-    "unbzip2-stream": "github:balena-io-modules/unbzip2-stream#4a54f56a25b58950f9e4277c56db2912d62242e7",
+    "unbzip2-stream": "^1.4.2",
```

Comment:
This version does not work.
The stream ends unexpectedly or detects the end of the stream wrong.
Causing issues like https://github.com/balena-io/etcher/issues/3901

Reply:
but isn't `4a54f56a25b58950f9e4277c56db2912d62242e7` > 5 years old?

Reply:
We should try to get our change(s) accepted upstream otherwise we own another module and have to maintain it.

Reply:
Converted our fixes to a simple git/diff patch, so we can archive this redundant GH repo:
https://github.com/balena-io-modules/unbzip2-stream once this PR merges.